TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -424,19 +424,13 @@ private static IAuthTransformerFactory buildAuthTransformerFactory(Parameters pa
             String serviceName = serviceAndRegion[0];
             String region = serviceAndRegion[1];
 
-            return new IAuthTransformerFactory() {
-                final DefaultCredentialsProvider defaultCredentialsProvider = DefaultCredentialsProvider.create();
-
-                @Override
-                public IAuthTransformer getAuthTransformer(IHttpMessage httpMessage) {
-                    return new SigV4Signer(defaultCredentialsProvider, serviceName, region, "https", null);
-                }
-
-                @Override
-                public void close() {
-                    defaultCredentialsProvider.close();
-                }
-            };
+            return new SigV4AuthTransformerFactory(
```

Comment:
Note to self - what does this factory look like, why IAuthTransformer?  Where does that interface live.